spec/admin.mediawiki

```diff
@@ -32,13 +32,19 @@ possible.
 {|
 ! variable !! units !! description
 |-
-| lot&nbsp;size  || atoms ||  the minimum order quantity and the order quantity increment when an asset is the base asset
+| lotSize  || int ||  the minimum order quantity and the order quantity increment when an asset is the base asset
 |-
-| rate&nbsp;step || atoms || the minimum price rate and the price rate increment when an asset is the quote asset. [[orders.mediawiki/#Rate_Encoding|message-rate encoding]]
+| rateStep || int || the minimum price rate and the price rate increment when an asset is the quote asset. [[comm.mediawiki/#Rate_Encoding|message-rate encoding]]
 |-
-| fee&nbsp;rate  || atoms/byte || the minimum fee rate for swap transactions
+| swapConf || int || the Minimum confirmations before acting on a swap transaction
 |-
-| swap&nbsp;confirmations|| count || the Minimum confirmations before acting on a swap transaction
+| maxFeeRate  || int || the maximum fee rate for swap transactions
+|-
+| network || string || the network the coin daemon is running one (i.e. "mainnet")
+|-
+| bip44symbol || string || the symbol of the [https://github.com/satoshilabs/slips/blob/master/slip-0044.md registered coin type]
+|-
+| configPath || string || the path to the coin daemon's config file or ipc file in the case of Ethereum
 |}
```

Comment:
This section was in the initial draft of the spec, before any code had been written. It's wasn't meant to describe a particular struct, but the motivation aligns most closely with `dex.Asset` rather than `AssetConf` because the specification describes only the API that servers implement, not implementation details. For example, we would never describe a variable called `configPath` in the spec, because it has nothing to do with how the server and client communicate.

The `AssetConf` info could be outlined in a wiki page or something, but not the spec.

Reply:
This is this the "Exchange Administration" section though, not client-server comms.

So while your comment about `AssetConf` being in wiki or somewhere else makes sense, this page actually does pertain to server config and administration details.

Reply:
> This is this the "Exchange Administration" section though, not client-server comms.

I just meant that this section isn't intended to describe implementation, just the relevant parameters for which the operator has knobs. The entire specification can be roughly defined as a definition of protocols for client-server comms, and that's the sense I was trying to use. The parameters listed in this section should eventually be communicated to the client, otherwise they don't really need to be listed.

Reply:
Gotcha, `configPath` is definitely not spec material, even in this section.

But spec/admin.mediawiki really does deviate from the proper scope of the spec as we understand it.  We have the asset params that are communicated defined in https://github.com/decred/dcrdex/blob/master/spec/fundamentals.mediawiki/#configuration-data-request.  So this Exchange Administration section really is somewhat tangential, and arguably out of place in the spec as a whole.

I just wanted to point out that the dex.Asset stuff is formalized in fundamentals.mediawiki, while this odd-ball section is actually aligned with AssetConf (and it does get uncomfortably close to the implementation).  I don't care too much what happens in this section as long as there's nothing grossly inaccurate.

Reply:
Uhh. Should I revert changes here and add it all to the wiki?

Reply:
I guess that's a buck question, but I'm fine keeping this admin stuff here without the configpath.